### ISSUE TITLE
update rats-pipelines use of app/services

### ISSUE THREAD

**Issue by ms-lolo** [microsoft/rats#120]
just need to update this component to use the new `rats.apps` package and create a new application for a pipeline lifecycle. i'll migrate the existing api with minimal changes to unblock other work and try to prototype a more generalizable approach to the main app execution in separate prs.

**Comment by ms-lolo** [comment]
i'm closing this in favor of #451 so we can keep our open tickets tied to more active development items. feel free to re-open this if someone finds time to re-introduce pipelines and get things to a functional state.
